ecl/eclccserver/eclccserver.cpp

```diff
@@ -854,14 +856,31 @@ int main(int argc, const char *argv[])
             if (optMonitorInterval)
                 startPerformanceMonitor(optMonitorInterval*1000, PerfMonStandard, nullptr);
 #ifdef _CONTAINERIZED
+            bool filtered = false;
+            std::unordered_map<std::string, bool> listenQueues;
+            Owned<IPTreeIterator> listening = globals->getElements("listen");
```

Comment:
Should there be a comment in a yaml file to indicate how these can be specified?

Reply:
There is, in values.yaml (I guess it got merged in a previous PR)

Reply:
Looks like it was added in

HPCC-23706 Add json schema to helm chart